### PR TITLE
Fix/docs errors

### PR DESCRIPTION
**Pull request by FLomb**
I have been fixing some things in the documentation and the GitHub homepage. First of all, some broken links to images pointing to the outdated `master` branch (now substituted by `main`). Second, some typos or other minor stylistic issues in the .rst files and jupyter notebooks of the docs.

I also wonder why we have the jupyter notebooks replicating the docs exactly. Are they necessary? I had to copy-and-paste any changes to the docs there, but I am not sure these notebooks are used anywhere.

### CODE REVIEW

**Comment by mohammadamint** [comment]
> I have been fixing some things in the documentation and the GitHub homepage. First of all, some broken links to images pointing to the outdated `master` branch (now substituted by `main`). Second, some typos or other minor stylistic issues in the .rst files and jupyter notebooks of the docs.
> 
> I also wonder why we have the jupyter notebooks replicating the docs exactly. Are they necessary? I had to copy-and-paste any changes to the docs there, but I am not sure these notebooks are used anywhere.

As .rst files are more compatible with Sphinx, it seems better to have the rst format of the Jupyter examples in the documentation, and having the jupyter files in the repo, could be also beneficial for the users as they can directly download them and run them. If something is fixed in the Jupyter files, the rst version can be downloaded without the need to replicate all the changes all over both Jupyter and .rsts.

**Review by mohammadamint — COMMENTED** [review]
In case that we are not sure, the changes are implemented both in notebooks and rst files, we can generate the rst files from Jupyter Notebook. Otherwise, we can continue already merge the PR!

File: docs/source/examples/simple_bulb/simple_bulb.rst [hunk 1]
```diff
@@ -153,7 +153,7 @@ Generating a profile for the ith day of the year
 .. image:: output_13_1.png
 
 
-whole year profile can be generated
+Whole year profile functionality
```
Comment (Bachibouzouk) [inline docs/source/examples/simple_bulb/simple_bulb.rst:156]: The ~ underneath the title should in principle, match the number of characters of the title

Comment (mohammadamint) [reply docs/source/examples/simple_bulb/simple_bulb.rst:156]: What do you think of having a github action to review the synatx issues in rst?

I found this library https://github.com/rstcheck/rstcheck that might be used to do so.

**Review by Bachibouzouk — APPROVED** [review]
Apart the title I have no other comments :)

File: docs/source/examples/simple_bulb/simple_bulb.rst (hunk 1 above)
Comment (FLomb) [reply docs/source/examples/simple_bulb/simple_bulb.rst:156]: I just fixed this issue; thanks for noticing. It'd be cool to have automated .rst syntax checks, I agree! But that's for a future PR

**Comment by FLomb** [comment]
@Bachibouzouk I am not quite sure how this is possible since I only touched the docs, but now we have some failing tests about the code. Namely: 
```
FAILED tests/test_calc_peak_time_range.py::test_peak_time_range_values - assert 0.03103792853653431 > 0.05
FAILED tests/test_switch_on.py::TestRandSwitchOnWindow::test_coincidence_normality_on_peak - AssertionError: The 'coincidence' values are not normally distributed.
```
Could it be that the 'randomness' embedded in these tests makes them occasionally not pass?

**Comment by mohammadamint** [comment]
> @Bachibouzouk I am not quite sure how this is possible since I only touched the docs, but now we have some failing tests about the code. Namely:
> 
> ```
> FAILED tests/test_calc_peak_time_range.py::test_peak_time_range_values - assert 0.03103792853653431 > 0.05
> FAILED tests/test_switch_on.py::TestRandSwitchOnWindow::test_coincidence_normality_on_peak - AssertionError: The 'coincidence' values are not normally distributed.
> ```
> 
> Could it be that the 'randomness' embedded in these tests makes them occasionally not pass?

To me this should be the case, as in PR #97, I am successfully passing the tests.

**Comment by FLomb** [comment]
I updated the PR with the fix that was preventing PR #97 from passing the tests. After updating, the issue with the failed tests above disappeared, reinforcing the idea that this is perhaps due to the randomness occasionally ending up out of the test bounds. Something to double-check!

**Pull request merged**